spreaded-secrets.yml

```diff
@@ -72,6 +72,12 @@ repositories:
     SONATYPE_USERNAME: SONATYPE_USERNAME
     SONATYPE_PASSWORD: SONATYPE_PASSWORD
     ADMIN_GITHUB_TOKEN: ADMIN_GITHUB_TOKEN
+  higherkindness/mu-scala.g8:
+    PGP_PASSPHRASE: PGP_PASSPHRASE
```

Comment:
I'm not sure if we need any secrets for this repo, apart from maybe the GitHub token? It's a g8 template, so we don't need to do any releases to Sonatype.

Reply:
👍  e7f8ecd